### PR TITLE
ExternalTrafficPolicy support before 4.10

### PR DESCRIPTION
**Pull request by Rupesh-git-eng**
The clusters are by default set up with ETP=local, but as the patch for local is yet not exist, we still balance the traffic even with ETP set to local. 
Post the patch release (4.10), we will support local. 

spec:
  clusterIP: 172.30.64.132
  externalTrafficPolicy: Local
  healthCheckNodePort: 32424

### CODE REVIEW

**Comment by Amrita42** [comment]
Related PR https://github.com/openshift/openshift-docs/pull/42558

**Comment by Amrita42** [comment]
@Rupesh-git-eng  Based on team feedback, the branch **openshift:4.9** is redundant. Please create your local branch from **enterprise-4.9**. Once you do that I have suggested the following based on your original text:

- Grammar change >> For the `ExternalTrafficPolicy` parameter, the default value is `local`. However the load is still distributed to the available backends until the fix is available, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1772847[BZ#1903408]. The value, `cluster`, is supported for both the parameters.`
- Tag QE >>https://github.com/zhaozhanqi
- Post QE review, please tag the peer-review squad 
- Also let me know if you have any queries , happy to help

File: modules/nw-ovn-kuberentes-limitations.adoc [hunk 1]
```diff
@@ -7,9 +7,13 @@
...
 The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has the following limitations:
 
-* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`.
-The default value, `cluster`, is supported for both parameters.
-This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
+* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`. The default value `cluster` is supported for both parameters. This limitation can affect you when you add a service of type LoadBalancer, NodePort, or add a service with an external IP.
+
+[NOTE]
+====
+Openshift Installer provisioned infrastructure(IPI) on Azure cloud, installer create Loadbalancer service with `ExternalTrafficPolicy` pointing to local, however the load is still distributed to the available backends until the fix is available, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1772847[BZ#1903408].
```
Comment (Miciah) [inline modules/nw-ovn-kuberentes-limitations.adoc:14]: It shouldn't matter whether it's IPI or UPI, and the issue would apply to other platforms as well as Azure.  

The link URL has one BZ number, and the link text has a different number.  I think you want the latter: <https://bugzilla.redhat.com/show_bug.cgi?id=1903408>.  

Is the intent here to add a BZ citation, or is there additional information that isn't covered by the existing note?

**Comment by Rupesh-git-eng** [comment]
Closing this pull request as the [new pull request](https://github.com/openshift/openshift-docs/pull/42584) has been forked from the enterprise-4.9 branch.

**Pull request closed without merging**